app/jobs/isolate_job.rb

```diff
@@ -196,4 +201,20 @@ def strip_output(output)
   rescue ArgumentError
     return output
   end
-end
+
+  def call_webhooks(submission)
+    webhooks = submission.webhooks
+    webhooks.each do | item |
+      url = item["url"]
+      data =  {
+        "postdata": item["postdata"],
+        "submission": submission.to_json
+      }
+      response = HTTParty.post(
```

Comment:
What happens if this doesn't succeed? Don't we want to log that in our database somehow?

Reply:
True, we definitely should. We shall start with logging errors/warnings here.